.azuredevops/pull_request_template.md

```diff
@@ -19,11 +19,21 @@ This pull request fulfills the following requirements :
 
 [comment]:# (Please strikethrough  non-applicable items \(https://docs.microsoft.com/en-us/azure/devops/project/wiki/markdown-guidance?view=azure-devops#emphasis-bold-italics-strikethrough\))
 
-- [ ] Interface members are XML documented.
-- [ ] Documentation (XML or comments) has been added or updated.
-- [ ] [Architecture documents](./doc/Architecture.md) have been updated.
+- [ ] Code documentation was updated.
+  - Interface members are XML documented.
+  - Comments or XML doc was updated.
```

Comment:
It's nothing but I would say `Comments or XML documentation was updated.` instead.